lib/screens/profile_screens/support/support_screen.dart

```diff
@@ -1,106 +1,121 @@
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: [
+                              Padding(
+                                padding: const EdgeInsets.only(right: 80.0),
+                                child: Text(
+                                  'If you have any questions or concerns, Please find our'.i18n,
+                                  style: Theme.of(context).textTheme.buttonLowEmphasis,
+                                ),
+                              ),
+                              Text(
+                                'Seeds Light Wallet',
+                                style: Theme.of(context).textTheme.headline8.copyWith(color: AppColors.canopy),
+                              ),
+                              const SizedBox(height: 8.0),
+                              RichText(
+                                text: TextSpan(
+                                  style: Theme.of(context).textTheme.buttonLowEmphasis,
+                                  children: <TextSpan>[
+                                    TextSpan(text: 'Channel in'.i18n),
+                                    TextSpan(
+                                        text: ' Discord ',
+                                        style: Theme.of(context)
+                                            .textTheme
+                                            .buttonLowEmphasis
+                                            .copyWith(color: AppColors.canopy)),
+                                    TextSpan(text: 'here.'.i18n),
+                                  ],
+                                ),
+                              )
+                            ],
                           ),
-                          const SizedBox(height: 8.0),
-                          RichText(
-                            text: TextSpan(
-                              style: Theme.of(context).textTheme.buttonLowEmphasis,
-                              children: <TextSpan>[
-                                TextSpan(text: 'Channel in'.i18n),
-                                TextSpan(
-                                    text: ' Discord ',
-                                    style: Theme.of(context)
-                                        .textTheme
-                                        .buttonLowEmphasis
-                                        .copyWith(color: AppColors.canopy)),
-                                TextSpan(text: 'here.'.i18n),
-                              ],
-                            ),
-                          )
-                        ],
-                      ),
+                        ),
+                      ],
                     ),
-                  ],
+                  ),
                 ),
-              ),
+                const Spacer(),
+                Text(state.appName ?? ""),
+                Text(state.version ?? ""),
+                Text(state.buildNumber ?? ""),
```

Comment:
I think Sky wants these 2 values in one line.

Reply:
![image](https://user-images.githubusercontent.com/51983923/153701507-c1805f3e-4d41-4b9c-84e9-e48c9dc62c2b.png)

Done